### PR TITLE
[SCB-1926] enhance CommonExceptionData to include more common data

### PR DESCRIPTION
**Pull request by wujimin**

### CODE REVIEW

File: swagger/swagger-invocation/invocation-core/src/test/java/org/apache/servicecomb/swagger/invocation/exception/CommonExceptionDataTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    assertThat(Json.encode(data)).isEqualTo("{\"code\":\"code\",\"message\":\"msg\"}");
+  }
+
+  @Test
+  void should_include_dynamic_field_in_json() {
+    CommonExceptionData data = new CommonExceptionData("msg");
+    data.putDynamic("k", "v");
+
+    assertThat(Json.encode(data)).isEqualTo("{\"message\":\"msg\",\"k\":\"v\"}");
```
Comment (liubao68) [inline swagger/swagger-invocation/invocation-core/src/test/java/org/apache/servicecomb/swagger/invocation/exception/CommonExceptionDataTest.java:46]: Can '{\"message\":\"msg\",\"k\":\"v\"}' decode to the same data?

Comment (wujimin) [reply swagger/swagger-invocation/invocation-core/src/test/java/org/apache/servicecomb/swagger/invocation/exception/CommonExceptionDataTest.java:46]: yes, i'll add one case

Comment (liubao68) [reply swagger/swagger-invocation/invocation-core/src/test/java/org/apache/servicecomb/swagger/invocation/exception/CommonExceptionDataTest.java:46]: Great

Comment (wujimin) [reply swagger/swagger-invocation/invocation-core/src/test/java/org/apache/servicecomb/swagger/invocation/exception/CommonExceptionDataTest.java:47]: done

**Pull request merged**